source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql

```diff
@@ -216,45 +216,23 @@ VALUES
-VALUES
+VALUES 
     (39, N'196e8946-6b41-4ca1-a2c9-7b1ae93cbf96', N'activity-view', N'196e8946-6b41-4ca1-a2c9-7b1ae93cbf96', N'Ability view activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
 
-INSERT [dbo].[PIMS_CLAIM]
+INSERT [dbo].[PIMS_CLAIM] 
     ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
-VALUES
+VALUES 
     (40, N'0db3f117-3da9-40bc-b5db-df676f40cc06', N'activity-add', N'0db3f117-3da9-40bc-b5db-df676f40cc06', N'Ability to add activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
 
-INSERT [dbo].[PIMS_CLAIM]
+INSERT [dbo].[PIMS_CLAIM] 
     ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
-VALUES
+VALUES 
     (41, N'7ebcb98d-f2fb-4826-a8e3-14729b95cc95', N'activity-edit', N'7ebcb98d-f2fb-4826-a8e3-14729b95cc95', N'Ability to edit activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
 
-INSERT [dbo].[PIMS_CLAIM]
+INSERT [dbo].[PIMS_CLAIM] 
     ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
-VALUES
+VALUES 
     (42, N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'activity-delete', N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'Ability to delete activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
-
--- Project claims
```

Comment:
@dfilteau concerned that it looks like the build for IS47 removes all of the project claims introduced in IS46.